README.md

```diff
@@ -25,7 +25,56 @@ with API token or Jenkins CLI access but can also appear with features
 such as the [Authorize Project
 plugin](http://plugins.jenkins.io/authorize-project/)
 or other plugins that require details of user permissions or group
-membership outside of a user's web session)
+membership outside of a user's web session).
+
+### Compatibility Notes
+
+Various LDAP servers use different operational attributes to make decisions on
+and expose configurations of concepts such as disabling an account, locking an
+account, and specifying a time interval the account is valid for. These policies
+are normally enforced by the LDAP server itself when performing user authentication.
+Once a user has configured API tokens or SSH keys, these parallel authentication
```

Comment:
This sentence needs better transition or explanation. The previous sentences are talking about LDAP servers, but this suddenly jumps, without mentioning it, to talking about Jenkins API tokens and SSH keys. The following sentence then goes back to talking about LDAP server implementation / configuration.

Maybe it needs to lead off with some sort of problem statement or explanation.

"Compatibility Notes" kind of makes sense, but only within a better context of what this whole section is about.